docs/tutorials/p2p.md

```diff
@@ -173,9 +154,9 @@ I have mine set up with two arguments: the first is the recipient as a string an
 === "Godot 2.x, 3.x"
 
 	```gdscript
-	func send_p2p_packet(this_target: int, packet_data: Dictionary) -> void:
+	func send_message(this_target: int, packet_data: Dictionary) -> void:
```

Comment:
I use a more updated version of this send message like this:
```
func send_packet(packet_data: Dictionary) -> void:
	var sending_packet: PackedByteArray = []
	sending_packet.append_array(var_to_bytes(packet_data).compress(FileAccess.COMPRESSION_GZIP))
	if is_player_host():
		for this_player in lobby_members.keys():
			if this_player != steam_id:
				if not Steam.sendMessageToUser(this_player, sending_packet, Steam.NETWORKING_SEND_RELIABLE, channel_client) == Steam.RESULT_OK:
					print("Failed to send Mesages host packet to %s: %s" % [this_player, sending_packet])
	else:
		if not Steam.sendMessageToUser(Steam.getLobbyOwner(lobby_id), sending_packet, Steam.NETWORKING_SEND_RELIABLE, channel_host) == Steam.RESULT_OK:
			print("Failed to send Messages client packet to %s: %s" % [Steam.getLobbyOwner(lobby_id), sending_packet])
```
Though I think some of it needs explained with updates to the lobby tutorial.